tutorials/commands/flux-proxy-command.rst

```diff
@@ -0,0 +1,194 @@
+.. _flux-proxy-command:
+
+=================
+Flux Proxy Basics
+=================
+
+With Flux, it is very common to create a Flux :ref:`subinstances<subinstance>`.  However, it can be confusing how to interact with those subinstances.
+
+This tutorial will introduce the :core:man1:`flux-proxy` command, which can be used to interact with subinstances.
+
+-----------------------
+Starting a Subinstance
+-----------------------
+
+First, let's create a subinstance to work with.  The most common way is through ``flux mini batch``.  We'll launch a subinstance and run a few jobs under that instance, we then call ``flux job status`` to wait on those jobs to finish.
+
+.. code-block:: sh
+
+   #!/bin/sh
```

Comment:
Wait where does flux mini batch come in? This looks like it's running 4x flux mini submit?

Reply:
To answer your question @v, `flux mini batch` is the `flux mini` utility that submits a "batch job" which in Flux is a job that runs an instance of Flux on the assigned resources with a batch script as its "initial program"

Reply:
okay - so you give it a script with other flux submit commands? As opposed to (what I've seen for other managers) a text file with some list of single commands (without the job manager directive) to run?

Reply:
How does that (conceptually) compare to a job array?

Reply:
> Wait where does flux mini batch come in? This looks like it's running 4x flux mini submit?

I listed the script and the run the `flux mini batch` right below, I guess I should put a sentence in between, perhaps its not obvious when skimming.